src/core/cardano/walletConnect/peerConnection.ts

```diff
@@ -0,0 +1,77 @@
+
+  private walletInfo = {
+    address: "",
+    name: "idw_p2p",
+    icon: PeerConnection.IDW_LOGO,
+    version: packageInfo.version,
+    requestAutoconnect: true,
+  };
+
+  private announce = [
+    "wss://tracker.openwebtorrent.com",
+    "wss://dev.tracker.cf-identity-wallet.metadata.dev.cf-deployments.org",
+    "wss://tracker.files.fm:7073/announce",
+    "ws://tracker.files.fm:7072/announce",
+    "wss://tracker.openwebtorrent.com:443/announce",
+  ];
+
+  private identityWalletConnect: IdentityWalletConnect;
+  private connected = false;
+
+  constructor() {
+    this.identityWalletConnect = new IdentityWalletConnect(
+      this.walletInfo,
+      localStorage.getItem("meerkat-identity-wallet-seed"),
```

Comment:
We should use the `Preferences` storage in the wallet instead of local storage. "meerkat-identity-wallet-seed" can also be made as a static readonly variable.